src/Dfc.CourseDirectory.WebV2/SharedViews/Components/ProviderTopNav.cshtml

```diff
@@ -41,29 +41,42 @@
                 Venues
             </a>
         </li>
-        @if (Model.ShowCoursesLinks && Model.ShowApprenticeshipsLinks)
+        @if (FeatureFlagProvider.HaveFeature(FeatureFlags.DataManagement) && ((Model.ShowCoursesLinks && Model.ShowApprenticeshipsLinks) || Model.ShowApprenticeshipsLinks || Model.ShowCoursesLinks))
```

Comment:
Should be displayed regardless of provider type
```suggestion
        @if (FeatureFlagProvider.HaveFeature(FeatureFlags.DataManagement))
```

Reply:
I've removed these now.